pkg/apps/application-list.jsx

```diff
@@ -47,7 +47,11 @@ const ApplicationRow = ({ comp, progress, progress_title, action }) => {
 
     let summary_or_progress;
     if (progress) {
-        summary_or_progress = <ProgressBar title={progress_title} data={progress} />;
+        summary_or_progress = (
+            <div className="sum-or-prog">
```

Comment:
This should be better replace with:

`<Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>`